pkg/client/config.go

```diff
@@ -79,45 +119,57 @@ func CheckTimeout(c context.Context, which *time.Duration, err error) error {
 }
```

Comment:
I can't comment on it for some reason, but the error message above should be "can be configured as timeouts.%s" right?

Reply:
Fixed!